SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -1649,4 +1649,18 @@ SequenceableCollection : Collection {
 	sanitize { arg ... args;
 		^this.multiChannelPerform(\sanitize, *args);
 	}
+
+	sortItemOccurrence {
+		var answer = this.asSet.asArray.collect { |uniqueItem, uniqueIndex|
+			Dictionary[
+				\uniqueItem -> uniqueItem,
+				\occurrence -> this.occurrencesOf(uniqueItem)
+			]
+		}.sortBy(\occurrence)
+		.reverse.collect { |item|
+			item.atAll([\uniqueItem, \occurrence]).asEvent
+		}
+		.do {|item| item };
```

Comment:
this does nothing ...

Reply:
Ah, that line was originally `.do {|item| item.postcs };`. So the result was:
```
(4: 4)
("3": 3)
('2': 2)
('a': 1)
-> [(4: 4), ("3": 3), ('2': 2), ('a': 1)]
```
by evaluating `[\a, \2, '2', "3", "3", "3", 4, 4, 4, 4, ].sortItemOccurrence`.
While removing `postcs` after @capital-G's suggestion, I falsely left `.do {|item| item }`.

I will remove the whole line.

Reply:
Done in https://github.com/supercollider/supercollider/commit/93ea5e552c73e3195f3a7f86185f5e61aea785e0